corehq/apps/hqwebapp/urls.py

```diff
@@ -92,7 +93,12 @@
     url(r'^relogin/$', login_new_window, name='login_new_window'),
     url(r'^relogin/iframe/$', iframe_domain_login_new_window, name='iframe_domain_login_new_window'),
     url(r'^log_email_event/(?P<secret>[\w]+)/?$', log_email_event, name='log_email_event'),
-
+    url(
+        r'^oauth/applications/register/',
+        OauthApplicationRegistration.as_view(),
+        name=OauthApplicationRegistration.urlname
+    ),
+    url(r'^oauth/', include('oauth2_provider.urls', namespace='oauth2_provider')),
```

Comment:
what are the implications of this moving here?

Reply:
The previous route redefines one of the paths from this package. Since that view is in `hqwebapp`, I felt it was best to place that route here instead of polluting the global urls file. I needed to put this line directly after so that it would check the routes in the right order.